### ISSUE TITLE
IPv4 addresses not parsed correctly in IIS

### ISSUE THREAD

**Issue by culyerr** [SpiderLabs/ModSecurity#1109]
Port number is included when passed to tree_contains_ip(..) and code attempts to parse as an IPv6 address.

Error is raised to Windows Event Log when doing IPmatch rules
```
[client 213.78.82.1:34984] ModSecurity: IPmatch: bad IPv6 specification "213.78.82.1:34984". [hostname "ACTUSAA1"] [uri "/api/ping"] [unique_id "16717361818946771401"]
```

**Comment by zimmerle** [comment]
This issue was resolved by #1110.

**Comment by zimmerle** [comment]
Related to: #734

**Comment by zimmerle** [comment]
Fixed by: #1220.
